Scan/CK3/prototype.kll

```diff
@@ -0,0 +1,73 @@
+Name = MD1;
```

Comment:
Also not necessary, it's just a defaultMap.kll for a very very early Infinity 60% that had a different matrix layout.